### PR TITLE
Fixes #3420

### LINKED ISSUE #3240: Manually Post processes entire download directory on TorrentToMedia call
Branch/Commit: SickRage@8965a3145b69c292e7c139ae065d5f8fa7c57dda
OS: Linux (ubuntu), Python 2.7

What you did: Tried to Post process a show

What happened: Sickrage continuously post-processes the show (yes actually copies continuously) while also manually scanning through every other file in the download directory.

What you expected: It to work like SickRage@a3ce318f2c3b207da474db4c8e3cf62bb9032e0e where it only processes the file/directory of the show we care about (so how its worked forever)

A few *notes*:

* Manual Post-processing is disabled (unticked) in `config/postProcessing`
* I left the `Post Processing Dir` field blank as well.
* I'm utilizing the latest nzbtoMedia (pulled today). This exact same behavior was happening with my previous version which was two years old.
* On torrent download, rtorrent launches calls `TorrentToMedia` to notify Sickrage about that particular download.

First off, here's the log from `TorrentToMedia`:
```
[17:44:58] [DEBUG]::SERVER: Attempting to connect to server at http://localhost:555/sickrage
[17:44:58] [DEBUG]::SERVER: Server responded at http://localhost:555/sickrage
[17:44:58] [INFO]::MAIN: SickBeard:tv fork set to sickrage
[17:44:58] [POSTPROCESS]::SICKBEARD: SUCCESS: The download succeeded, sending a post-process request
[17:44:58] [DEBUG]::SICKBEARD: Opening URL: http://localhost:555/sickrage/home/postprocess/processEpisode with params: {'nzbName': 'show.mkv', 'quiet': 1, 'failed': 0, 'force_next': 1, 'proc_dir': '/rtor/rtorrent/downloads/tv'}
```

Right when that first call is made, here's what I see from Sickrage (reverse order; latest on top):

```
2017-02-27 17:45:00 DEBUG Thread-17 :: Processing folder /****/****rent/downloads/tv
2017-02-27 17:45:00 DEBUG Thread-17 :: Processing folder /****/****rent/downloads/tv
2017-02-27 17:45:00 INFO Thread-17 :: Beginning manual post processing task: /****/****rent/downloads/tv
```

And then a crap load of scanning logs as it goes through everything in the `tv/` directory for absolutely no reason. Additionally, it **re-copies the newly added episode continuously** being sure to also shoot out notifications (i.e. pushover) so for one episode, I get repeated notifications to my phone.

As I know there was a big rewrite of the Post-processing system, I'm guessing this call needs to be modified somehow? I'm guessing the value of `proc_dir` is being used as a literal scanning location? Some input and help here would be appreciated ;)

### PR DESCRIPTION
**Pull request by miigotu**
min_max for COMING_EPS_MISSED_RANGE

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by sharkykh** [comment]
Umm, by default check_setting_int would return the default value:
```
:param fallback_def: if True, `def_val` will be returned when value not in range of `min_val` and `max_val`.
                     otherwise, `min_val`/`max_val` value will be returned respectively (default: True)
```
You need to set it to False for it to work like min_max.

**Comment by miigotu** [comment]
? You mean where I min_max in the save_config?

File: sickbeard/__init__.py [hunk 1]
```diff
@@ -1358,7 +1358,7 @@ def path_leaf(path):
         COMING_EPS_LAYOUT = check_setting_str(CFG, 'GUI', 'coming_eps_layout', 'banner')
         COMING_EPS_DISPLAY_PAUSED = check_setting_bool(CFG, 'GUI', 'coming_eps_display_paused')
         COMING_EPS_SORT = check_setting_str(CFG, 'GUI', 'coming_eps_sort', 'date')
-        COMING_EPS_MISSED_RANGE = check_setting_int(CFG, 'GUI', 'coming_eps_missed_range', 7)
+        COMING_EPS_MISSED_RANGE = check_setting_int(CFG, 'GUI', 'coming_eps_missed_range', 7, min_val=0, max_val=42810)
```
Comment (sharkykh) [inline sickbeard/__init__.py:1361]: Right here,
```python
COMING_EPS_MISSED_RANGE = check_setting_int(CFG, 'GUI', 'coming_eps_missed_range', 7, min_val=0, max_val=42810, fallback_def=False)
```

Comment (sharkykh) [reply sickbeard/__init__.py:1361]: **`min_max`** returns minimum value if `val < min_val`, and maximum value if `val > max_val`
**`check_setting_int/float`** only does that when `fallback_def = False`

Comment (miigotu) [reply sickbeard/__init__.py:1361]: Either way is ok to me, I just dont want it throwing errors lol

Comment (miigotu) [reply sickbeard/__init__.py:1361]: Change it for me in your next pr =P movie time with the kids

Comment (sharkykh) [reply sickbeard/__init__.py:1361]: :+1: njoy

**Pull request merged**